ui/eureka/eureka.js

```diff
@@ -1144,6 +924,34 @@ let Options = {
         en: 'Pyros Elemental',
         cn: '涌火元灵',
       },
+      bunnys: {
+        northbunny: {
+          label: {
+            en: 'Bunny',
```

Comment:
I wonder if it'd be worth defining a bunnyLabel object above, and then just having `label: bunnyLabel` to not have to repeat this.

Reply:
added.